test/unit/module/real/algorithm/all/regular/all.hpp

```diff
@@ -137,11 +143,18 @@ TTS_CASE_TPL("Check eve::all[ignore]", EVE_TYPE)
 
         TTS_EXPECT_NOT(eve::all[eve::ignore_extrema_(i, j)](mask));
         TTS_EXPECT(eve::all[eve::ignore_extrema_(i + 1, j)](mask));
+
+        TTS_EXPECT_NOT(eve::all[eve::keep_between(i, T::static_size - j)](mask));
+        TTS_EXPECT(eve::all[eve::keep_between(i + 1, T::static_size - j)](mask));
+
         mask.set(i, true);
 
         mask.set(T::static_size - j - 1, false);
         TTS_EXPECT_NOT(eve::all[eve::ignore_extrema_(i, j)](mask));
```

Comment:
ignore_extrema_ is supposed to be generated by `&&`. Do you think it is helpful we have it as a 1st class citizen?